test/src/main/scala/scala/test/twitter_scrooge/bare_jar_test/BUILD

```diff
@@ -0,0 +1,21 @@
+load("//scala:scala.bzl", "scala_binary")
+load("//thrift:thrift.bzl", "thrift_library")
+load("//twitter_scrooge:twitter_scrooge.bzl", "scrooge_scala_library")
+
+java_import(
+    name = "barejar",
+    jars = ["bare-thrift.jar"]
+)
+
+scrooge_scala_library(
+    name = "bare_thrifts",
+    visibility = ["//visibility:public"],
+    remote_self_jars = [":barejar"]
```

Comment:
can we put this on `thrift_library` instead? That makes it clearer the jar is only thrift and really has nothing to do with scrooge.